src/sage/matrix/matrix2.pyx

```diff
@@ -13611,8 +13611,9 @@ cdef class Matrix(Matrix1):
                 if max_location != -1:
                     perm[k], perm[max_location] = perm[max_location], perm[k]
                     M.swap_rows(k, max_location)
+                    inv = M.get_unsafe(k, k).inverse()
                     for j in range(k+1, m):
-                        scale = -M.get_unsafe(j, k)/M.get_unsafe(k, k)
+                        scale = -M.get_unsafe(j, k)*inv
```

Comment:
Would even be better if space around `*` were respected :-)
```diff
-                        scale = -M.get_unsafe(j, k)*inv
+                        scale = -M.get_unsafe(j, k) * inv
```

Reply:
thanks for the suggestion